### ISSUE TITLE
Can't import { Quill } from Typescript

### ISSUE THREAD

**Issue by JohnStrick** [zenoamaro/react-quill#361]
The issue I'm having is that I can't import { Quill } from react-quill using TypeScript. Here's the code snippet:

`import ReactQuill, { Quill } from 'react-quill';`

Typescript reports the following error:
`[ts] Module <path to module>' has no exported member 'Quill'.

Looking at the typings file, I don't see any export for the Quill object. I believe this is what is causing the issue.

#### React-Quill version 
1.2.7

#### tsconfig.json
```
{
  "compilerOptions": {
    "outDir": "./dist/",
    "sourceMap": true,
    "allowJs": true,
    "checkJs": false,
    "module": "esnext",
    "target": "esnext",
    "jsx": "react",
    "moduleResolution": "node",
    "baseUrl": ".",
    "allowSyntheticDefaultImports": true,
  },
  "include": ["**/*.ts", "**/*.tsx", "typings/*.d.ts"]
}
```

**Comment by JohnStrick** [comment]
The fix for this seems pretty easy. I created a PR to add Quill as an exported object to the typings file.

**Comment by skiant** [comment]
Heya! Thanks for the fix, I was running into the same issues.

Do you think you could push a release to NPM to fix the issue? 
A direct install from the git url doesn't seem to work at the moment.

**Comment by alexkrolick** [comment]
Published in 1.3.0
